apps/gateway/lib/gateway/champions_socket_handler.ex

```diff
@@ -121,6 +126,16 @@ defmodule Gateway.ChampionsSocketHandler do
     end
   end
 
+  defp handle(%GetAfkRewards{user_id: user_id}),
+    do: prepare_response(%{afk_rewards: Users.get_afk_rewards(user_id)}, :afk_rewards)
+
+  defp handle(%ClaimAfkRewards{user_id: user_id}) do
+    {:ok, afk_rewards} =
```

Comment:
We shouldn't do these kinds of matches here. This will result in a raise if Users.claim_afk_rewards fails! Which could happen since reset_afk_rewards_claim returns either {:ok, updated_value} or {:error, reason}. Let's handle this appropiately and match the result like our other handles